bedrock/firefox/views.py

```diff
@@ -555,14 +557,23 @@ def new(request):
                 template = 'firefox/new/portland/scene1-fast.html'
             elif experience == 'portland-safe':
                 template = 'firefox/new/portland/scene1-safe.html'
+            elif experience == 'forgood':
+                context['portland_variation'] = 'portland-v-forgood'
+                template = 'firefox/new/portland/scene1.html'
+            elif experience == 'fast':
+                context['portland_variation'] = 'portland-fast-v-fast'
+                template = 'firefox/new/portland/scene1-fast.html'
+            elif experience == 'safe':
+                context['portland_variation'] = 'portland-safe-v-safe'
+                template = 'firefox/new/portland/scene1-safe.html'
             else:
                 template = 'firefox/new/scene1.html'
         else:
             template = 'firefox/new/scene1.html'
 
     # no harm done by passing 'v' to template, even when no experiment is running
     # (also makes tests easier to maintain by always sending a context)
-    return l10n_utils.render(request, template, {'v': variant})
+    return l10n_utils.render(request, template, context)
```

Comment:
We could be a bit cleaner by passing the pre-defined `experience` in `context`:

`return l10n_utils.render(request, template, {'experience': experience})`

This would eliminate the rather repetitive calls if the `elif` chain, and give us consistency in the JS selectors (which would need updating along with the variable name in the templates).